xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -4218,11 +4216,11 @@ void CDVDPlayer::GetAudioStreamInfo(int index, SPlayerAudioStreamInfo &info)
   }
 }
 
-int CDVDPlayer::AddSubtitleFile(const std::string& filename, const std::string& subfilename, CDemuxStream::EFlags flags)
+int CDVDPlayer::AddSubtitleFile(const std::string& filename, const std::string& subfilename)
```

Comment:
No, it doesn't access an internal accesses demuxer. But it's altering m_SelectionStreams. So should I add a message?